docs/source/guide/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -187,14 +147,62 @@ The number of the subset datasets is 8, 16, and 24.
 | Lite-HRNet-x-mod3  | 50.23 | 50.93   | 60.09 | 61.61   | 62.66 | 64.87   |
 +--------------------+-------+---------+-------+---------+-------+---------+
 
-You can use Self-supervised learning like the below command.
+Unlike other tasks, two things are considered to use self-supervised learning:
+
+- `--train-data-root` must be set to a directory only containing images, not ground truths.
+  DetCon uses pseudo masks created in `detcon_mask` directory for training. If they are not created yet, they will be created first.
+- `--val-data-root` is not needed.
```

Comment:
Same with the above comments. need to use the explicit expression.

Reply:
We decided to keep it as it is.